applications/crossbar/doc/tasks.md

```diff
@@ -110,11 +119,15 @@ curl -v -X GET \
 }
 ```
 
+#### Add a new system task
+
 
 #### Add a new task
 
 > PUT /v2/accounts/{ACCOUNT_ID}/tasks
 
+Note: There are tasks that run against system resources, only for use by the superduper admin (like rate uploading), which can omit `/accounts/{ACCOUNT_ID}` from the URI. Leaving the account in the URI should have no impact.
+
```

Comment:
1. "super admin is a crossbar construct" this is false. "cb_tasks takes care of sending the auth account id to new/7 when there is no account id on the url path" your code does account_id = auth_account_id = cb_context:auth_account_id, which violates the concept.
2. Either you justify a usage for it, either you remove `/v2/tasks`